multirange.js

```diff
@@ -59,8 +59,38 @@ self.multirange = function(input) {
 		ghost.style.setProperty("--high", input.valueHigh * 100 / max - 1 + "%");
 	}
 
-	input.addEventListener("input", update);
-	ghost.addEventListener("input", update);
+    function passClick(evt) {
+      // Are the ghost and input elements inverted? (ghost is lower range)
```

Comment:
Please match the code style of the rest of the project, including using tabs for indentation and spaces only for alignment.